components/sup/doc/http_gateway_butterfly_schema.json

```diff
@@ -59,7 +114,7 @@
                   },
                   "status": {
                     "description": "The status of the election",
-                    "type": "string"
+                    "enum": ["Running", "NoQuorum", "Finished"]
```

Comment:
At some point, I'd like to revisit the `Display` implementations of this enum, as well as that of health below... these are upper-case, whereas some of the other enums below are lowercase. Nothing that needs to be fixed right now, but something to keep in mind for #5692.

Reply:
I'll make a note of it on that issue.